dz19/src/index.jsx

```diff
@@ -0,0 +1,9 @@
+import { createRoot } from 'react-dom/client'
+import Body from './App.jsx'
```

Comment:
```suggestion
import App from './App.jsx'
```

для дефолтный импортов используй названия соответствующе названию файлов - это упрощает понимание где и что, когда импортов и файлов в проекте много